repose-aggregator/components/filters/versioning/pom.xml

```diff
@@ -63,14 +63,20 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <scope>test</scope>
+        </dependency>
         <dependency>
-            <groupId>org.codehaus.jackson</groupId>
-            <artifactId>jackson-core-lgpl</artifactId>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
             <scope>test</scope>
```

Comment:
this isn't going to be respected unless it's being dependency managed at test in the parent chain. If it's truly meant to be test only here, it needs to also be dependency managed to test here.